### ISSUE TITLE
Non-preview extension reports "Preview"

### ISSUE THREAD

**Issue by dbaileyut** [PowerShell/vscode-powershell#2774]
### System Details

System Details Output
```
### VSCode version: 1.46.0 a5d1cc28bb5da32ec67e86cc50f84c67cc690321 x64

### VSCode extensions:
amazonwebservices.aws-toolkit-vscode@1.10.0
DavidAnson.vscode-markdownlint@0.36.0      
donjayamanne.githistory@0.6.5
DotJoshJohnson.xml@2.5.0
eamodio.gitlens@10.2.2
ecmel.vscode-html-css@0.2.3
emilast.LogFileHighlighter@2.8.0
HookyQR.beautify@1.5.0        
humao.rest-client@0.24.1      
jakebathman.mysql-syntax@1.3.1
ms-dotnettools.csharp@1.22.1
ms-dotnettools.vscode-dotnet-runtime@0.1.2
ms-mssql.mssql@1.9.0
ms-python.python@2020.6.88468
ms-vscode-remote.remote-wsl@0.44.3
ms-vscode.cpptools@0.28.3
ms-vscode.powershell@2020.6.0
msazurermtools.azurerm-vscode-tools@0.10.0
pejmannikram.vscode-auto-scroll@1.0.3
Shan.code-settings-sync@3.4.3
streetsidesoftware.code-spell-checker@1.9.0


### PSES version: 2.2.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.19041.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

### Issue Description

The production release includes "(Preview)" references when it is not preview.

#### Expected Behaviour

The extension metadata should not not indicate it is Preview

#### Actual Behaviour

The production release appears to be preview

![VSCodePSExtension-ReportsPreview](https://user-images.githubusercontent.com/7095040/85046282-d3161e00-b155-11ea-8d1d-ab1e44c0d468.png)
Via browser:
![VSCodePSExtension-ReportsPreview2](https://user-images.githubusercontent.com/7095040/85046291-d6a9a500-b155-11ea-9c0f-cbff73f8e43f.png)

### Attached Logs

Follow the instructions in the [troubleshooting docs](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#logs)
about capturing and sending logs.

**Comment by SydneyhSmith** [comment]
@dbaileyut thanks for opening this, luckily this is purely cosmetic but we will make sure to fix the error in the next release

**Comment by PSVortex** [comment]
I also have this issue.

**Comment by corbob** [comment]
Would it be possible to release another stable version? Or at least re-release 2020.6.0 without the preview tag? I'm still running into people mentioning that they're on the preview version when they're on the stable version (or asking which preview version they should be on).

**Comment by JeremyTBradshaw** [comment]
I was here today to open an issue for this, searched and found this already opened issue.  So will just add that I too notice the issue and it caused me to take extra time figuring out what version I was on when getting support for another issue.

**Comment by SydneyhSmith** [comment]
Thanks everyone, we will be sure to get this right on our next release (hopefully coming this month)

**Comment by andschwa** [comment]
This was resolved in #3207
